examples/custom_configuration_auto_register/lib/entities.rb

```diff
@@ -0,0 +1,6 @@
+require 'import'
+
+class Entities
+  include Import['persistence.db']
+
+end
```

Comment:
It'd be more representative of a real app if we had `lib/entities/user.rb` instead of `lib/entities.rb`. Would you mind doing that?